addons/l10n_no/data/account_tax_data.xml

```diff
@@ -34,6 +34,7 @@
         <record id="tax2" model="account.tax.template">
             <field name="chart_template_id" ref="no_chart_template"/>
             <field name="name">1 Inngående mva høy sats 25%</field>
+            <field name="sequence">0</field>
```

Comment:
@rgo-odoo You can put it as sequence 0, but also put it first in the xml-file please.  Did you check sales side as well?

Reply:
@jco-odoo sales has default 25% already.